devel/libbacktrace/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        ianlancetaylor libbacktrace 7939218
+
+checksums           rmd160  112d148c86fa1a19a2bf5284e74f5241308d8b76 \
+                    sha256  858b1225351052234f7e71f0f36622fc9ad33aac947db7816d0b443ae0dd33ce \
+                    size    624629
```

Comment:
I don’t know if defaults changed again, but at least until recently it was advised to set `github.tarball_from` explicitly.